common/djangoapps/course_action_state/admin.py

```diff
@@ -6,4 +6,15 @@
 
 from common.djangoapps.course_action_state.models import CourseRerunState
 
-admin.site.register(CourseRerunState)
+
+class CourseRerunStateAdmin(admin.ModelAdmin):
+
+    exclude = ["course_key"]
+    def get_queryset(self, request):
+        # For any query against this table, remove course_key field
+        # because that field might have bad data in it.
+        qs =CourseRerunState.objects.defer('course_key')
```

Comment:
Extra space here needed after the `=` sign. Are we concerned with this introducing any latency whatsoever? Is it better to delete the row instead, is what I'm getting to.

Reply:
I am not concerned with latency with this change. I believe if this change works, the change to db should be immediate.
I think we should try to merge this PR, let me self-serve, before we ask you to delete from the stage environment directly.